templates/comments/list.html

```diff
@@ -94,7 +94,19 @@ <h2><i style="padding-right: 0.3em" class="fa fa-comments"></i>{{ _('Comments')
                             <div class="content content-description">
                                 <div class="comment-title"{% if not node.title or node.title.startswith('Re:') %}
                                      style="display: none"{% endif %}>{{ node.title }}</div>
-                                <div class="comment-body">
+                                    {% if node.score > vote_hide_threshold %}
```

Comment:
How about you show `real-comment-body` unconditionally, and add `style="display: none"` as an inline `if`? Then, conditionally display `bad-comment-body`. This way, the indentation actually lines up properly.

Alternatively, use CSS to hide the `real-comment-body`:

```css
.bad-comment-body + .real-comment-body {
    display: none;
}
```